tensorbay/opendataset/CCPDGreen/catalog.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Do CCPDGreen and CCPD has same catalog?
If yes, I think it will be better to put `def CCPDGreen` and `def CCPD` in the same file

Reply:
yes, they have same catalog. But on our website, they come from different pages.
https://gas.graviti.cn/dataset/data-decorators/CCPD
https://gas.graviti.cn/dataset/data-decorators/CCPDGreen